src/common/commonutils/DeviceInfoUtils.c

```diff
@@ -287,8 +287,13 @@ bool IsCpuFlagSupported(const char* cpuFlag, void* log)
 
     if ((NULL != cpuFlag) && (NULL != strstr(cpuFlags, cpuFlag)))
     {
+        OsConfigLogInfo(log, "CPU flag'%s' is supported", cpuFlag);

```

Comment:
nit: should there be a space here: `flag '%s'`

Reply:
Good catch